src/common/transformers/GET.js

```diff
@@ -0,0 +1,20 @@
+import _get from 'lodash.get';
+import { transformerWithArg } from './transformer';
+
+export const get = (value, path) => {
+    if (!value) {
```

Comment:
not needed as _.get accept a default return value as the third parameter.

Reply:
ok, this test is useless, i will drop it